torch/onnx/_internal/fx/passes/decomp.py

```diff
@@ -47,17 +49,27 @@ def _run(self, *args, **kwargs) -> torch.fx.GraphModule:
         #  mul = sym_size_2 * sym_size_3;  sym_size_2 = sym_size_3 = None
         #  view: f32[3, 5, 20] = torch.ops.aten.view.default(x, [sym_size, sym_size_1, mul])
 
-        fx_mode = "symbolic" if self.enable_dynamic_axes else "fake"
+        fake_mode: Any = self.fake_mode
+        maybe_fake_args = self._maybe_fakefy_args(fake_mode, *args)
+        if fake_mode is not None:
+            # Using existing fake mode as context, signal `make_fx` that it does not need
+            # to create a new fake mode by passing tracing_mode as "real".
+            tracing_mode = "real"
```

Comment:
This also makes `make_fx` create a graph with static shape.
https://github.com/pytorch/pytorch/issues/105464 has a task to track fake mode + dynamic shape, but a TODO would make finding this integration point easier in the future

Reply:
This does not make static shape.

This allows reusing the dynamo administrative fake mode, which contains dynamic shape info if dynamic shape is turned on.